packages/utils/recursive-parser.test.js

```diff
@@ -2,41 +2,98 @@
 
 const defineTest = require("./defineTest");
 
-defineTest(__dirname, "init", "fixture-1", "entry", {
-	objects: "are",
-	super: [
-		"yeah",
-		{
-			test: new RegExp(/\.(js|vue)$/),
-			loader: "'eslint-loader'",
-			enforce: "'pre'",
-			include: ["customObj", "'Stringy'"],
-			options: {
-				formatter: "'someOption'"
+
+defineTest(
+	__dirname,
+	"init",
+	"fixture-1",
+	"entry",
+	{
+		objects: "are",
+		super: [
+			"yeah",
+			{
+				test: new RegExp(/\.(js|vue)$/),
+				loader: "'eslint-loader'",
+				enforce: "'pre'",
+				include: ["customObj", "'Stringy'"],
+				options: {
+					formatter: "'someOption'"
+				}
 			}
-		}
-	],
-	nice: "':)'",
-	foo: "Promise.resolve()",
-	man: "() => duper"
-});
-
-defineTest(__dirname, "add", "fixture-2", "entry", {
-	objects: "are not",
-	super: [
-		"op",
-		{
-			test: new RegExp(/\.(wasm|c)$/),
-			loader: "'pia-loader'",
-			enforce: "'pre'",
-			include: ["asd", "'Stringy'"],
-			options: {
-				formatter: "'nao'"
+		],
+		nice: "':)'",
+		foo: "Promise.resolve()",
+		man: "() => duper"
+	}
+);
+
+defineTest(
+	__dirname,
+	"add",
+	"fixture-2",
+	"entry",
+	{
+		objects: "are not",
+		super: [
+			"op",
+			{
+				test: new RegExp(/\.(wasm|c)$/),
+				loader: "'pia-loader'",
+				enforce: "'pre'",
+				include: ["asd", "'Stringy'"],
+				options: {
+					formatter: "'nao'"
+				}
 			}
-		}
-	],
-	nice: "'=)'",
-	foo: "Promise.resolve()",
-	man: "() => nice!!",
-	mode: "super-man"
-});
+		],
+		nice: "'=)'",
+		foo: "Promise.resolve()",
+		man: "() => nice!!",
+		mode: "super-man"
+	}
+);
+
+defineTest(
+	__dirname,
+	"remove",
+	"fixture-3",
+	"entry",
+	{
+		a: null,
+	}
+);
+
+defineTest(
+	__dirname,
+	"remove",
```

Comment:
You can just do one deep test for remove with multiple props

Reply:
This approach is non-recursive and it doesn't drill down deep inside props.
Recursive drilling is only needed for the creation of nodes.

We do cover every case as demonstrated in the GIF. :)
The tests have to be individual. We can add more tests if needed. I think one-two test for each prop type should be good.